util/steambox

```diff
@@ -51,9 +51,14 @@ declare -a DRIDEVS=()
 if [[ -d /dev/dri ]] ; then
 	DRIDEVS+=(/dev/dri/*)
 fi
-if [[ -d /dev/ati ]] ; then 
+if [[ -d /dev/ati ]] ; then
 	DRIDEVS+=(/dev/ati/*)
 fi
+for cdev in nvidiactl nvidia0 nvidia-modeset; do
```

Comment:
@rsharo I think a match on additional `nvidia` nodes makes good sense.  And thank you for the advice on updating a PR; I was just this morning thinking it very clunky to have to close and open additional requests to follow up on commentary.

I will push this update shortly.